src/nft/pages/collection/index.tsx

```diff
@@ -121,90 +121,94 @@ const Collection = () => {
-                  width="full"
-                  src={
-                    collectionStats?.bannerImageUrl
-                      ? `${collectionStats.bannerImageUrl}?w=${window.innerWidth}`
-                      : undefined
-                  }
-                  className={styles.bannerImage}
-                  background="none"
-                />
-              </BannerWrapper>
-              <CollectionDescriptionSection>
-                {collectionStats && (
-                  <CollectionStats stats={collectionStats || ({} as GenieCollection)} isMobile={isMobile} />
-                )}
-                <div id="nft-anchor" />
-                <ActivitySwitcher
-                  showActivity={isActivityToggled}
-                  toggleActivity={() => {
-                    isFiltersExpanded && setFiltersExpanded(false)
-                    toggleActivity()
-                  }}
-                />
-              </CollectionDescriptionSection>
-              <CollectionDisplaySection>
-                <Box
-                  position={isMobile ? 'fixed' : 'sticky'}
-                  top={{ sm: '0', md: '72' }}
-                  left="0"
-                  width={isMobile ? 'full' : '0'}
-                  height={isMobile && isFiltersExpanded ? 'full' : undefined}
-                  background={isMobile ? 'backgroundBackdrop' : undefined}
-                  zIndex="modalBackdrop"
-                  overflowY={isMobile ? 'scroll' : undefined}
-                >
-                  {isFiltersExpanded && (
-                    <>
-                      {isMobile && (
-                        <MobileFilterHeader>
-                          <ThemedText.HeadlineSmall>Filter</ThemedText.HeadlineSmall>
-                          <IconWrapper onClick={() => setFiltersExpanded(false)}>
-                            <BagCloseIcon />
-                          </IconWrapper>
-                        </MobileFilterHeader>
-                      )}
-                      <Filters traitsByGroup={collectionStats?.traits ?? {}} />
-                    </>
+            loading ? (
+              <CollectionPageSkeleton />
+            ) : (
+              <>
+                <BannerWrapper width="full">
+                  <Box
+                    as={collectionStats?.bannerImageUrl ? 'img' : 'div'}
+                    height="full"
+                    width="full"
+                    src={
+                      collectionStats?.bannerImageUrl
+                        ? `${collectionStats.bannerImageUrl}?w=${window.innerWidth}`
+                        : undefined
+                    }
+                    className={styles.bannerImage}
+                    background="none"
+                  />
+                </BannerWrapper>
+                <CollectionDescriptionSection>
+                  {collectionStats && (
+                    <CollectionStats stats={collectionStats || ({} as GenieCollection)} isMobile={isMobile} />
```

Comment:
do you need to do the || {} condition here? looks like if `collectionStats` is defined it should be `GenieCollection`